component/org.wso2.carbon.identity.oauth2.grant.token.exchange/src/main/java/org/wso2/carbon/identity/oauth2/grant/token/exchange/internal/TokenExchangeComponentServiceHolder.java

```diff
@@ -58,17 +58,6 @@ public void setApplicationManagementService(ApplicationManagementService applica
         this.applicationManagementService = applicationManagementService;
     }
 
-    public Collection<UserOperationEventListener> getUserOperationEventListenerCollection() {
-
-        return userOperationEventListenerCollection;
-    }
-
-    public void setUserOperationEventListenerCollection(Collection<UserOperationEventListener>
```

Comment:
Why are these methods removed? was this introduced by you for the same feature?

Reply:
Yes, this was introduced in the previous PR. There was a comment to make some improvements by not maintaining two data structures. Since that pr was merged because of the urgency i'm addressing those comments in this pr